extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/config/VaultRuntimeConfig.java

```diff
@@ -150,6 +150,12 @@
     @ConfigItem
     public Map<String, CredentialsProviderConfig> credentialsProvider;
 
+    /**
+     * transit config
+     */
+    @ConfigItem
```

Comment:
I think we should probably have a proper title in the javadoc here and a `@ConfigDocSection` annotation.